src/coinbase/staking_operation.ts

```diff
@@ -202,4 +250,30 @@ export class StakingOperation {
       }
     }
   }
+
+  /**
+   * loadTransactionsFromModel loads new unsigned transactions from the model into the transactions array.
+   * Note: For External Address model since tx signing and broadcast status happens by the end user and not our backend
+   * we need to be careful to not overwrite the transactions array with the response from the API. Ex: End user could have used
+   * stakingOperation.sign() method to sign the transactions, and we should not overwrite them with the response from the API.
+   * This however is ok to do so for the Wallet Address model since the transactions are signed and broadcasted by our backend.
```

Comment:
To my knowledge the signing still happens on the client side but only broadcasting happens in the backend.

Reply:
Sry can you point which part of the above comment you disagree with ? it's a long one 😅

Reply:
Aah got it. Good catch. Reworded it completely. Take a look.